### ISSUE TITLE
Feature request: Add body crop

### ISSUE THREAD

**Issue by ayalalazaro** [lassoan/SlicerTotalSegmentator#19]
TotalSegmentator and SlicerTotalSegmentator are awesome, congratulations for the great job!  We are currently testing it for autosegmentation in radiation oncology. We use 8GB GPUs which sometimes fails to allocate all the memory with big studysets. It turns out that the TotalSegmentation option '--body_seg' makes the process error free and takes less time as well. 
We have modified SlicerTotal to always use this option, can it be added to the extension maybe just as the --fast option is currently implemented?

**Comment by lassoan** [comment]
> can it be added to the extension maybe just as the --fast option is currently implemented?

Exactly. Would you like to give it a try to implement it? You can add the checkbox in Qt designer and get the checkbox value the same way as it is done for the fast mode checkbox.

**Comment by ayalalazaro** [comment]
I am fairly new to 3d slicer, but I can give it a try, sure!

-------- Mensaje original --------

**Comment by lassoan** [comment]
Maybe you don't need it (because the work is mostly doing the same what is done for the fast mode), but if you were interested, here is a programming introduction tutorial for Slicer: https://github.com/PerkLab/PerkLabBootcamp/blob/master/Doc/day3_2_SlicerProgramming.pptx
